website/src/app/[lang]/[region]/(website)/partners/[slug]/page.tsx

```diff
@@ -3,9 +3,17 @@ import { PartnerHome } from '@/app/[lang]/[region]/(website)/partners/(component
 import { NgoEntryJSON } from '@/app/[lang]/[region]/(website)/partners/(types)/PartnerCards';
 import { Translator } from '@socialincome/shared/src/utils/i18n';
 import { redirect } from 'next/navigation';
+import { ngos } from '../(sections)/ngolist';
 
-function getNGOTranslation(translator: Translator, slug: string): NgoEntryJSON {
-	return translator.t(slug);
+function getNGOTranslation(translator: Translator, slug: string): NgoEntryJSON | undefined {
```

Comment:
This works, but is a bit weird imo. It would be much cleaner in my opinion to have all ngos in the translation file in one object and then translate that entire object. The object with the translations of all ngs could then be passed to the PartnerHome component.